explainer.md

```diff
@@ -362,21 +363,147 @@ merchants that accept credit cards.
 
 ## Security Considerations
 
-### Enrollment in cross-origin iframes
+On top of the [WebAuthn security considerations], there are a few
+considerations specific to this proposal.
 
 ### Cross-origin authentication ceremony
 
-### Merchant-supplied data
+One of the features that SPC adds is the ability for a third-party (the
+Merchant) to utilize credentials provided by the Relying Party (the Account
+Provider), to provide low-friction authentication for payments. This can
+expose the Relying Party to both login and payment attacks.
+
+**Login attack**
+
+In this attack, a malicious third-party uses SPC (with some previously-obtained
+credentials for an identified user, either legimitately or illicitly) to obtain
+a payment assertion. They then send that assertion to the Relying Party's
+**login** end-point, and hope that the Relying Party does not follow the
+[WebAuthn requirements on assertion verification][webauthn-verification].
+
+SPC does make sure that payment assertions differ from login assertions, in the
+following ways:
+
+1. The CollectedClientData `type` member is "payment.get", not "webauthn.get"
+1. The CollectedClientData `origin` member will be the calling origin, which
+     would **not** be the Relying Party in such an attack.
+1. The CollectedClientData will also have an additional `payment` member for
+     an SPC-generated assertion.
+
+In addition to the above, a Relying Party should also record what type of
+interaction (i.e. login or payment) a given `challenge` is generated for and
+ensure that the use of the assertion matches the expected interaction type.
+
+**Payment attack**
+
+In this attack, a malicious third-party tries to use SPC (again with either
+legimitately or illictly obtained credentials) to initiate an unauthorized
+payment. The attack is infeasible:
```

Comment:
Editorial: Change "The attack is infeasiable" to "Such an attack has a low chance of success for several reasons:"